package.json

```diff
@@ -80,7 +80,7 @@
     "replace-special-characters": "^1.2.5",
     "rimraf": "^3.0.2",
     "rxjs": "^7.3.0",
-    "sqlite3": "^5.0.2",
+    "sqlite3": "^4.2.0",
```

Comment:
Do we even need sqlite nowadays?

Reply:
Removed in 443557f76bd044a67aec07fadcdf76ac30e40f97